impl/pom.xml

```diff
@@ -95,12 +95,12 @@
 
         <dependency>
             <groupId>org.jboss.spec.javax.resource</groupId>
-            <artifactId>jboss-connector-api_1.6_spec</artifactId>
+            <artifactId>jboss-connector-api_1.7_spec</artifactId>
         </dependency>
 
         <dependency>
             <groupId>org.jboss.spec.javax.jms</groupId>
-            <artifactId>jboss-jms-api_1.1_spec</artifactId>
+            <artifactId>jboss-jms-api_2.0_spec</artifactId>
```

Comment:
Shouldn't we get rid of these jobs spec in favor of jakarta ones?

Reply:
I don't know. I don't recall the original reason why we added these instead of javax. I think they are supersets of the original APIs or just 1:1. And they are in Central so I see no harm in them.